src/mergeTests.ts

```diff
@@ -325,6 +322,43 @@ function insertAnnotations(
   }
 }
 
+/**
+ * get type of import.
+ * returns 0 for static improts
+ * 2 for static java imports
+ * 3 for java imports
+ * and 1 for other importsk
+ *
```

Comment:
Multiple typos and stuff.

```suggestion
 * Get type of import
 *
 * Returns:
 *   0 for static imports
 *   2 for static java imports
 *   3 for java imports
 *   1 for other imports
```